clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheStateServiceImpl.java

```diff
@@ -242,6 +251,15 @@ private ResourcePageSource createPageSource(String poolName, ServerSideConfigura
     return pageSource;
   }
 
+  private void registerStatistic(Object context, String alias, String observerName, Callable callable) {
+    Set<String> tags = new HashSet<String>(Arrays.asList("ClusteredServerSide","tier"));
```

Comment:
I just updated it. I added a constant for "PageSource" and removed "ClusteredServerSide"